Signal/src/util/DisplayableText.swift

```diff
@@ -11,13 +11,83 @@ import Foundation
     let fullText: String
     let displayText: String
     let isTextTruncated: Bool
+    let jumbomojiCount: NSNumber?
```

Comment:
Is there a better way to do an objective-c compatible optional number in Swift?

Reply:
Since Objc doesn't have optional primitive types, this seems like the right way to do that.

But in looking how we're using it, does it even benefit from being optional? 

In all the cases we're returning nil, it seems like zero is actually the right semantics. e.g. There are 0 jumbo-emoji in the empty string. There are 0 jumbo-emoji in a really long string. etc.

Reply:
Great point.